### ISSUE TITLE
Pressing options don't work until after some tries

### ISSUE THREAD

**Issue by Fskadi** [transmission/transmission#7106]
### What is the issue?

When selecting properties of a torrent or open in folder, the action happens only after doing the action repeatedly, sometimes as much as 10 times till it happens, i have SSD i don't think is a speed issue, as it takes sometimes more than a few seconds(tries)

### Which application of Transmission?

None

### Which version of Transmission?

downloaded from flatpak,last version

**Comment by mikedld** [comment]
Unless you can reproduce with non-Flatpak version, or at least specify the actual client type and version, I'm inclined to close this. We're not supporting "none" applications here. Reproducing with non-Flatpak version would still be helpful as we're not releasing these packages ourselves.

**Comment by Fskadi** [comment]
4.0.5 (a6fe2a64aa) is the version said in about, i will migrate to bittorrent, i wanted to let you know about the issue tho

**Comment by Fskadi** [comment]
actually, i like transmission, the only thing that i miss is minimizing in system tray but i'm in no hurry, so beside the version, i will download the non-flatpak version and see how it works

**Comment by ckerr** [comment]
Closing due to age + lack of actionable info + lack of response to requested information.

If this issue persists for anyone in >= 4.1.0, please open a new ticket, preferably with the information that @mikedld requested above. Thank you!
